_admin/setup/LDAP-config-AD.md

```diff
@@ -6,32 +6,22 @@ last_updated: tbd
 
 
 ## Before you begin
-Before you configure LDAP for Active Directory, collect the following information:
-- **URL**: This is needed to connect to Active Directory. For example, `ldap://192.168.2.48:389`
-- **Domain name**: Preferred domain under which users who want to be authenticated against Active Directory reside. When a user logs in with a username, the default domain is added to the username before sending it to the LDAP server. If users reside in multiple domains, you can still designate one of them as the default.
-
-  **_NOTE:_** Users belonging to a non-default domain will have to explicitly qualify their username when they log in, for example: `username@ldap1.thoughtspot.com`
-- **Search base**: LDAP search base that allows ThoughtSpot to find user properties such as email and display name from LDAP.
-- **SSL**: Obtain the SSL certificate from an issuing authority if you want to use SSL.
-- **Automatically add LDAP or AD users in ThoughtSpot? (yes/no)**: If you choose 'yes', the user that does not exist in ThoughtSpot is automatically created and authenticated against LDAP. Note that such users passwords exist only in LDAP and are not stored in ThoughtSpot.
-
-  If you choose 'no', users have to be manually created with a dummy password as a placeholder in ThoughtSpot before they can log in. The username you specify when creating the LDAP authenticated user manually in ThoughtSpot has to be domain qualified, for example: `username@ldap1.thoughtspot.com`.
-
-  **_NOTE:_** In order to log in to ThoughtSpot, the user has to exist in ThoughtSpot independent of whether that user is authenticated against LDAP or against ThoughtSpot's internal authentication.
-- **Also use ThoughtSpot internal authentication? (yes/no)**: If you choose 'yes', ThoughtSpot will first attempt to authenticate the user against LDAP when a user logs in. If that attempt fails, it will then attempt to authenticate the user against ThoughtSpot. If either of these succeed, then the user is successfully logged in.
-
-  **_NOTE:_** You can use this option only in `tscli`. This is useful in scenarios where some users are not in LDAP and are created only in ThoughtSpot.
-
+Before you configure ThoughtSpot for Active Directory, collect the following information:
+- **URL**: This is needed to connect to Active Directory. For example, `ldap://ad.yourdomain.local:389` or `ldap://ad.yourdomain.local:636`
+- **Domain name**: Default domain under which users who want to be authenticated against Active Directory reside. When a user logs in with a username, the default domain is added to the username before sending it to the LDAP server. If users reside in multiple domains, you can still designate one of them as the default. **_NOTE:_** Users belonging to a non-default domain will have to explicitly qualify their username when they log in, for example: `username@ad.yourdomain.local`
```

Comment:
> If users reside in multiple domains, you can still...

This has to be multiple sub-domains. We do not support authenticating against multiple domains. We support authentication against multiple subdomains Ex: `na.thoughtspot.com`, `au.thoughtspot.com` etc but not `thoughtspot.com` and `ts.com`.